core/backend-itwin-client/src/oidc/AgentAuthorizationClient.ts

```diff
@@ -119,7 +98,7 @@ export class AgentAuthorizationClient extends BackendAuthorizationClient impleme
   /** Returns a promise that resolves to the AccessToken of the currently authorized client.
    * The token is refreshed if necessary.
    */
-  public async getAccessToken(requestContext?: ClientRequestContext): Promise<AccessToken> {
+  public async getAccessToken(requestContext?: ClientRequestContext): Promise<AccessTokenString> {
     if (this.isAuthorized)
       return this._accessToken!;

```

Comment:
could you update this so it doesnt use non null assertion

Reply:
Fixed.